.github/renovate.json5

```diff
@@ -33,6 +33,7 @@
     "renovate/stop-updating",
   ],
   "stopUpdatingLabel": "renovate/stop-updating",
+  "automerge": true,
```

Comment:
Let's hope that setting `automerge` globally will also enable it for all `packageRules` :pray: